changelogs/8.0.asciidoc

```diff
@@ -3,9 +3,24 @@
 
 https://github.com/elastic/apm-server/compare/7.13\...master[View commits]
 
+* <<release-notes-8.0.0-beta1>>
 * <<release-notes-8.0.0-alpha2>>
 * <<release-notes-8.0.0-alpha1>>
 
+[float]
+[[release-notes-8.0.0-beta1]]
+=== APM version 8.0.0-beta1
+
+https://github.com/elastic/apm-server/compare/v8.0.0-alpha2\...v8.0.0-beta1[View commits]
+
+[float]
+==== Added
+- Add support for CloudWatch metric streams through firehose endpoint {pull}6380[6380]
```

Comment:
Can we add an experimental batch here please?

Reply:
Yup!

Reply:
```suggestion
- experimental:[] Add support for CloudWatch metric streams through firehose endpoint {pull}6380[6380]
```